cfgov/jinja2/v1/wagtail/molecules/related-posts-half-width.html

```diff
@@ -0,0 +1,54 @@
+{# ==========================================================================
+
+   related-posts-half-width
+
+   ==========================================================================
+
+   Description:
+
+   Creates related posts markup in half width:
+
+   value:                  An object used to customize the markup.
+
+   value.related_posts:    An array of related posts.
+
+   value.view_more:        An link used to view more related posts.
+
+   ========================================================================== #}
+
+{% import 'macros/time.html' as time %}
+
+<div class="m-related-posts
+            m-related-posts__half-width">
```

Comment:
That makes more sense. In a previous incarnation of the related posts backend, it was a structblock so this made more sense to use. Thanks for flagging it.